x/ecocredit/client/core/proposal.go

```diff
@@ -0,0 +1,78 @@
+package core
+
+import (
+	"fmt"
+	"strconv"
+
+	govclient "github.com/cosmos/cosmos-sdk/x/gov/client"
+	"github.com/cosmos/cosmos-sdk/x/gov/client/rest"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	"github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	"github.com/regen-network/regen-ledger/x/ecocredit/core"
+)
+
+var CreditTypeProposalHandler = govclient.NewProposalHandler(TxCreditTypeProposalCmd, func(context client.Context) rest.ProposalRESTHandler {
+	return rest.ProposalRESTHandler{ // TODO: what to do here??
```

Comment:
anyone know what to do with this? couldnt find any other impls of this, and looks like its not there anymore on sdk master

Reply:
We are still supporting legacy rest. I think it is okay to return an error.

Reply:
We are still supporting legacy rest. I think it is okay to return an error.

Reply:
In Regen ledger? If we return an error then there is no point to keep this code.

Reply:
if we return an error, the CLI part still works - i assume returning an error here would be similar to returning http 501

Reply:
probably fine to not implement